src/components/material/MaterialHeader.tsx

```diff
@@ -74,6 +81,10 @@ const MaterialHeader: React.FC<MaterialHeaderProps> = ({
   const coverPid =
     (manifestation?.pid as Pid) || getManifestationPid(manifestations);
 
+  const faustId = manifestation
+    ? convertPostIdToFaustId(manifestation?.pid as Pid)
```

Comment:
Please remove the null chaining if it is not necessary. 

As far as I can tell we know that `manifestation` will always be set in this codebranch.